timesketch/frontend-ng/src/components/Visualization/VisualizationEditor.vue

```diff
@@ -36,9 +36,20 @@ limitations under the License.
       </v-btn>
     </v-toolbar>
     <v-divider class="mx-3"></v-divider>
+    <v-row class="mt-3">
+      <v-col >
+        <v-container class="ma-0">
+          <ts-timeline-search 
+            componentName="visualization" 
+            :analyzerTimelineId="selectedTImelineIDs"
```

Comment:
Loading a sketch with this PR throws the following error on the console:

```
[Vue warn]: Property or method "selectedTImelineIDs" is not defined on the instance but referenced during render. Make sure that this property is reactive, either in the data option, or for class-based components, by initializing the property. See: https://v2.vuejs.org/v2/guide/reactivity.html#Declaring-Reactive-Properties.

found in

---> <TsVisualizationEditor> at src/components/Visualization/VisualizationEditor.vue
       <Visualization> at src/views/Visualization.vue
         <Canvas> at src/views/Canvas.vue
           <VMain>
             <Sketch> at src/views/Sketch.vue
               <VApp>
                 <App> at src/App.vue
                   <Root>
```
Deleting the line should fix it, it was used to pre-select timelines based from the Timeline submenu. We can add that later for visualizations.

```suggestion
```